docs/rules/aws_iam_access_key_forbidden.md

```diff
@@ -0,0 +1,29 @@
+# aws_iam_access_key_forbidden
+
+Disallow using `aws_iam_access_key` resources.
+
+## Example
+
+```hcl
+resource "aws_iam_access_key" "my_key" {
+  user = "my-user"
+}
+```
+
+```
+$ tflint
+1 issue(s) found:
+
+Error: "aws_iam_access_key" resources are forbidden. (aws_iam_access_key_forbidden)
+
+  on main.tf line 1:
+   1: resource "aws_iam_access_key" "my_key" {
+```
+
+## Why
+
+Managing IAM access keys with terraform leads to sensitive secrets being stored in plain text in terraform state files.
```

Comment:
It seems that the secret is not included in the state when imported.

>  This attribute is not available for imported resources.

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_access_key#secret

For example, it may be difficult to enforce this rule in use cases where you manually create IAM keys, import them, and manage them with Terraform. What do you think?